cmd/lncli/commands.go

```diff
@@ -1822,6 +1830,7 @@ func listChainTxns(ctx *cli.Context) error {
 	if ctx.IsSet("end_height") {
 		req.EndHeight = int32(ctx.Int64("end_height"))
 	}
+	req.FilterTxHash = ctx.StringSlice("filter_tx_hash")
```

Comment:
nit: could add this directly to the struct initialization on L1825.

Reply:
Done